DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Delegates/ViewDelegateControllerTests.cs

```diff
@@ -47,5 +55,167 @@ public void Deactivating_delegate_returns_redirect()
             // then
             result.Should().BeRedirectToActionResult();
         }
+
+        [Test]
+        public void Removal_confirmation_page_displays_for_valid_delegate_and_course()

```

Comment:
Seeing as the logic for some of this can be moved out into the ServiceFilters, the tests can be removed that test that removed logic.